index.html

```diff
@@ -308,20 +308,155 @@
 
     <section class="informative">
       <h2>Introduction</h2>
+
+      <p>
+The use of shared secrets for [=authentication=] and [=authorization=], such as
+the use of passwords, has resulted in a variety of security failures over the
+past several decades. To address these security failures, systems can upgrade to
+the use of
+<a href="https://en.wikipedia.org/wiki/Public-key_cryptography">
+asymmetric cryptography</a>, which uses digital signatures that are far more
+difficult to compromise. However, one shortcoming of digital signatures is
+the difficulty in disseminating the information, such as public cryptographic
+keys, to those that would need to verify the security of the digital signature.
+      </p>
```

Comment:
Hmm, the first sentence here seems to beg for a "citation needed" response and a debate over the various reasons some shared secrets are better than others (or even better than asymmetric crypto under the right circumstances and so on) and then the comment about "past several decades" which will potentially become dated as this spec ages into the potentially "post quantum age" and so on. I'd rather we just highlight the advantages of asymmetric crypto:

```suggestion
Digital signatures, based on
<a href="https://en.wikipedia.org/wiki/Public-key_cryptography">asymmetric
cryptography</a>, can be used in [=authentication=] and [=authorization=]
schemes to make them difficult for adversaries to compromise. However,
one shortcoming of digital signatures is the challenge in disseminating
necessary information, such as public cryptographic keys, to those who need
to verify the security of a digital signature.
      </p>
```